src/widgetastic_patternfly5/components/drawer.py

```diff
@@ -0,0 +1,33 @@
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import View
+
+
+class BaseDrawer:
+    """Represents drawer component for pf5
+
+    https://www.patternfly.org/components/drawer
+    """
+
+    CLOSE = ".//button[@class='pf-v5-c-button pf-m-plain']"
```

Comment:
Today only we decided that give comp to both pf4 and pf5. 
So I would request you to remove `pf-v5` and just go with `".//button[@class='-c-button pf-m-plain']"`

same for others in the widget file. In the test file, you can go with a specific locator.

Reply:
hi @digitronik I tried using` .//button[@class='-c-button pf-m-plain']"` but the test is failing

Reply:
@yash2189 it doesn't work because you specifically looked for `-c-button` but you need to use `contains`
`.//button[contains(@class, 'c-button')]`

but in this case - it's better to use `aria-label`, so we are good

Reply:
Thanks @LightOfHeaven1994 yes I did use contains method but thought to use aria label since I thought it would make it much easier :)